### PR TITLE
Add types and default values references to `AlertDialog`, `AnimatedSwitcher`, `AppBar`, `Audio`, `AudioRecorder`, `AutoComplete`, `AutoFillGroup`, `Badge`, `View`, `WebView` and `WindowDragArea` controls

### PR DESCRIPTION
**Pull request by taaaf11**
Add types and default values references for `AlertDialog`, `AnimatedSwitcher`, `AppBar`, `Audio`, `AudioRecorder`, `AutoComplete`, `AutoFillGroup`, `Badge`, `View`, `WebView` and `WindowDragArea` controls.

### CODE REVIEW

**Comment by taaaf11** [comment]
@ndonkoHenri kindly review this pr

**Comment by taaaf11** [comment]
Ping @ndonkoHenri

**Comment by ndonkoHenri** [comment]
Did you see my comments?

**Review by ndonkoHenri — CHANGES_REQUESTED** [review]

File: docs/controls/alertdialog.md [hunk 1]
```diff
@@ -140,11 +144,13 @@ A control that is displayed at the top of the dialog. Typically a [`Icon`](/docs
...
 Padding around the `icon`.
 
+Value is of type `PaddingValue`.
+
 ### `inset_padding`
 
 Padding around the Dialog itself.
 
-The value is an instance of [`Padding`](/docs/reference/types/padding) class or a number.
+The value is of type [`PaddingValue`](/docs/reference/types/aliases#paddingvalue).
```
[the remark this answers is not in the document]
Comment (taaaf11) [reply docs/controls/alertdialog.md:153]: I have followed the style in these lines. No need to change here

**Comment by taaaf11** [comment]
Ping @ndonkoHenri

**Comment by FeodorFitsner** [comment]
@taaaf11 don't spam with pings please. @ndonkoHenri will take a look when available.

**Comment by taaaf11** [comment]
Sorry, will take care next time @FeodorFitsner

File: docs/controls/appbar.md [hunk 2]
```diff
@@ -176,11 +192,13 @@ Value is of type [`TextStyle`](/docs/reference/types/textstyle).
...
 
-Defaults to `56.0`.
+Value is of tye [`OptionalNumber`](/docs/reference/types/aliases#optionalnumber) and defaults to `56.0`.
 
 ### `toolbar_opacity`
 
-The opacity of the toolbar. Value ranges from `0.0` (transparent) to `1.0` (fully opaque) which is the default.
+The opacity of the toolbar. Value ranges from `0.0` (transparent) to `1.0` (fully opaque).
+
+VAlue is of type [`OptionalNumber`](/docs/reference/types/aliases#optionalnumber) and defaults to `1.0`.
```
Comment (ndonkoHenri) [inline docs/controls/appbar.md:201]: Value

File: docs/controls/badge.md [hunk 3]
```diff
@@ -68,45 +68,54 @@ For example:
 ```python
 badge.alignment = ft.alignment.top_left
 ```
+
+The value is of type [`Alignment`](/docs/reference/types/alignment)
```
Comment (ndonkoHenri) [inline docs/controls/badge.md:72]: ~The~

File: docs/controls/badge.md [hunk 4]
```diff
@@ -68,45 +68,54 @@ For example:
...
-A child Control contained by the badge, typically an icon that's part of a [`NavigationBar`](/docs/controls/navigationbar) or a [`NavigationRail`](/docs/controls/navigationrail) destination.
+A child `Control` contained by the badge, typically an icon that's part of a [`NavigationBar`](/docs/controls/navigationbar) or a [`NavigationRail`](/docs/controls/navigationrail) destination.
+
+Value is of type `Control`.
 
 ### `label_visible`
 
 If False, label is not displayed. By default, `label_visible` is True. It can be used to create a badge that's only shown under certain conditions.
 
+Valeu is of type `bool`.
```
Comment (ndonkoHenri) [inline docs/controls/badge.md:88]: Typo in Value

File: docs/controls/badge.md [hunk 5]
```diff
@@ -68,45 +68,54 @@ For example:
...
+
 ### `large_size`
 
 The badge's label height if `text` is provided.
 
-Default value is 16. If the default value is overridden then it may be useful to also override `padding` and `alignment`.
+If the default value is overridden then it may be useful to also override `padding` and `alignment`.
+
+Value is of type [`OptionalNumber`](/docs/reference/types/aliases#optionalnumber)  and default value is 16.
```
Comment (ndonkoHenri) [inline docs/controls/badge.md:96]: "... and defaults to `16`"

File: docs/controls/badge.md [hunk 6]
```diff
@@ -68,45 +68,54 @@ For example:
...
+Value is of type [`OffsetValue`](/docs/reference/types/aliases#offsetvalue).
 
 ### `padding`
 
 The padding added to the badge's label.
 
 This value is only used if `text` is provided. Defaults to 4 pixels on the left and right.
 
-Padding value is an instance of [`Padding`](/docs/reference/types/padding) class.
+Padding value is an instance of [`PaddingValue`](/docs/reference/types/aliases#paddingvalue)
```
Comment (ndonkoHenri) [inline docs/controls/badge.md:112]: "Value is of type ..."

File: docs/controls/view.md [hunk 7]
```diff
@@ -99,25 +105,35 @@ to `CrossAxisAlignment.START`.
 
 ### `on_scroll_interval`
 
-Throttling in milliseconds for `on_scroll` event. Default is `10`.
+Throttling in milliseconds for `on_scroll` event. Value is of type `int` and defaults to `10`.
```
Comment (ndonkoHenri) [inline docs/controls/view.md:108]: Can the typing-highlight be on a new line?

File: docs/controls/view.md [hunk 8]
```diff
@@ -99,25 +105,35 @@ to `CrossAxisAlignment.START`.
...
 
 ### `padding`
 
-A space between page contents and its edges. Default value is 10 pixels from each side.
+A space between page contents and its edges.
+
+Value is of type [`PaddingValue`](/docs/reference/types/aliases#paddingvalue) and defaults to `10`.
```
Comment (ndonkoHenri) [inline docs/controls/view.md:114]: "and defaults to `padding.all(10)`"

File: docs/controls/view.md [hunk 9]
```diff
@@ -99,25 +105,35 @@ to `CrossAxisAlignment.START`.
...
+
+Value is of type `str`
 
 ### `scroll`
 
 Enables a vertical scrolling for the Page to prevent its content overflow.
 
 Property value is an optional [`ScrollMode`](/docs/reference/types/scrollmode) enum with `None` as default.
 
+Value is of type [`ScrollMode`](/docs/reference/types/scrollmode) and defaults to `ScrollMode.None`
```
Comment (ndonkoHenri) [inline docs/controls/view.md:128]: ~" and defaults to `ScrollMode.None`"~

`ScrollMode.None` doesn't exist.

File: docs/controls/webview.md [hunk 10]
```diff
@@ -48,14 +48,20 @@ Set the background [color](/docs/reference/colors) of the WebView.
 
 Enable or disable the JavaScript execution on the page. Note, that disabling the JavaScript execution on the page may result to unexpected web page behaviour.
 
+Value is of type `bool`
```
Comment (ndonkoHenri) [inline docs/controls/webview.md:51]: Please add . to the end of all the sentences

**Review by ndonkoHenri — CHANGES_REQUESTED** [review]

File: docs/controls/alertdialog.md [hunk 11]
```diff
@@ -120,18 +122,22 @@ Value is of type [`ClipBehavior`](/docs/reference/types/clipbehavior) and defaul
 
 The (optional) content of the dialog is displayed in the center of the dialog in a lighter font. Typically this is a [`Column`](/docs/controls/column) that contains the dialog's [`Text`](/docs/controls/text) message.
 
+Value is of type [`Control`].
```
[the remark this answers is not in the document]
Comment (taaaf11) [reply docs/controls/alertdialog.md:125]: there is no doc for `Control` in the docs

[the remark this answers is not in the document]
Comment (ndonkoHenri) [reply docs/controls/alertdialog.md:125]: Then you can remove the square brackets []

[the remark this answers is not in the document]
Comment (taaaf11) [reply docs/controls/alertdialog.md]: removed

File: docs/controls/view.md [hunk 12]
```diff
@@ -125,7 +127,7 @@ Enables a vertical scrolling for the Page to prevent its content overflow.
 
 Property value is an optional [`ScrollMode`](/docs/reference/types/scrollmode) enum with `None` as default.
 
-Value is of type [`ScrollMode`](/docs/reference/types/scrollmode) and defaults to `ScrollMode.None`
+Value is of type [`ScrollMode`](/docs/reference/types/scrollmode) and defaults to `None`.
```
Comment (ndonkoHenri) [inline docs/controls/view.md:130]: If the default is `None`, then we don't mention.

File: docs/controls/view.md [hunk 13]
```diff
@@ -125,7 +127,7 @@ Enables a vertical scrolling for the Page to prevent its content overflow.
 
 Property value is an optional [`ScrollMode`](/docs/reference/types/scrollmode) enum with `None` as default.
```
Comment (ndonkoHenri) [inline docs/controls/view.md:128]: I guess we can remove this line.

**Review by ndonkoHenri — CHANGES_REQUESTED** [review]

File: docs/controls/view.md (hunk 13 above)
Comment (taaaf11) [reply docs/controls/view.md:128]: oh

File: docs/controls/alertdialog.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ndonkoHenri) [reply docs/controls/alertdialog.md:155]: @taaaf11

**Review by ndonkoHenri — APPROVED** [review]

**Pull request merged**